wbooks/src/app/components/AppNavigator/constants.ts

```diff
@@ -0,0 +1,22 @@
+import Navbar from '@components/Navbar';
+import { COLORS } from '@constants/colors';
+
+export const SCREEN_OPTS = {
+  headerTitleStyle: {
+    fontSize: 16,
+    color: COLORS.white,
+    fontWeight: '600',
+    letterSpacing: -0.4,
+    marginBottom: 20
+  },
+  headerLeftContainerStyle: {
+    marginBottom: 50
+  },
```

Comment:
Why this?

Reply:
Because otherwise i can't lower the header margin. We've looked up with Maxi about this to no answer. Maybe we can talk later about this?

Reply:
Ok, never mind. It's fine for me